nx.json

```diff
@@ -3,52 +3,48 @@
     "default": {
       "runner": "nx/tasks-runners/default",
       "options": {
-        "cacheableOperations": []
+        "cacheableOperations": [
+          "cached:build:esbuild",
+          "cached:build:definitions",
+          "cached:build:doc:extract",
+          "cached:build:doc",
+          "cached:build:locales",
+          "cached:build:stencil",
+          "cached:build:storybook:assets-list",
+          "cached:build:storybook",
+          "cached:validate:storybook",
+          "cached:validate:definitions",
+          "cached:test:unit"
+        ]
```

Comment:
Unfortunately, based on Nx's documentation, it looks like all cacheable operations need to be defined at the root `nx.json`.

I added the `cached:` prefix to cachable tasks for the following reasons:
* Viewable in projects
  * While modifying/refactoring task definitions, it's very important to know which tasks are cached and which aren't.
* Searchability
  * It's not particularly intuitive when reading `project.json` files that some tasks are cached at all, let alone how to make a task cacheable. With this change, the `cached:` prefix is a bit of a red flag. When somebody searches `cached:`, they'll find `cacheableOperations`.
* Separation
  * Both the `atomic` and `atomic-hosted-page` projects have a `build:stencil` task. However, we don't want to cache `atomic-hosted-page`'s (yet).